kpet/schema.py

```diff
@@ -396,13 +396,20 @@ class List(Node):
     """
     List schema, with every element matching a single specified schema.
     """
-    def __init__(self, element_schema):
+    def __init__(self, element_schema, min_len=None):
```

Comment:
Could you use zero as the default?

Reply:
Please add an `assert(type(min_len) is int)` and an `assert(min_len >= 0)`.

Reply:
Also, please add the docstring for the constructor.

Reply:
Ah, of course, I was thinking of the `max_len` I guess :sweat_smile: 

I'll fix this and the rest, thanks!